### ISSUE TITLE
Press Ctrl+Z will always remain 1 character

### ISSUE THREAD

**Issue by verils** [cmderdev/cmder#1617]
When I typed a wrong command and I tried to use `ctrl+z` to go back, there is always the first character of the command line remained there.

Like this:
![default](https://user-images.githubusercontent.com/14834095/34931474-f74b32fc-fa09-11e7-82de-697da3d8d27b.png)

I can see this character, but it does not exist actually, because I can not keep typing the rest of command, I need to type that first character again. 

The result is, the command line looks so weird.

**Comment by Stanzilla** [comment]
Hey there,
Cmder uses ConEmu as terminal emulator, if you are experiencing a problem with that, please open a bug over at https://github.com/Maximus5/ConEmu

Thank you! :)

**Comment by verils** [comment]
Sorry, but it seems that ConEmu does not support pressing `Ctrl + Z` to revoke.

I downloaded ConEmu for latest release version(v18.01.14), and checked its hotkey settings, there was no `Ctrl + Z` binding. Then I pressed this key in ConEmu, it went to be like this:

![default](https://user-images.githubusercontent.com/14834095/35324594-2203c724-012c-11e8-9071-041ebf6c8c0a.png)

It behaved as same as Windows CMD does.

![default](https://user-images.githubusercontent.com/14834095/35324965-5150f438-012d-11e8-9bb1-43ba733f8829.png)

I don't know why and how Cmder supports revoking. Maybe it is a system feature? I don't think ConEmu's manager can give any help for me.
